src/models/todo-item.model.ts

```diff
@@ -0,0 +1,48 @@
+import { Entity, property, model } from '@loopback/repository';
+import { SchemaObject } from '@loopback/openapi-spec';
+
+@model()
+export class TodoItem extends Entity {
+  @property({
+    type: 'number',
+    id: true
+  })
+  id: number;
```

Comment:
a new line here to visually separate the property definitions would help :-)

Reply:
Not sure if you meant for this comment to be down a line or not, please look again at the changes and see if it's what you were looking for.

Reply:
It is definitely what I'm looking for 8)